docs/source/concept_models/vivarium_swissre_stomachcancer/concept_model.rst

```diff
@@ -155,7 +157,10 @@ Scale-up of stomach cancer screening using ABC method with endoscopic follow-up
 
 :underline:`Alternative scenario`
 
-In the alternative scenario, there will be a scale up of ABC screening starting from 5% to 30% as indicated in the coverage figure below. We do not assume that the 5% who recieve H. pylori screening in the baseline is the same groups those who recieve screening in the alternative scenario. 
+In the alternative scenario, there will be a scale up of ABC screening starting from 5% to 30% as indicated in the coverage figure (orange line) below. 
```

Comment:
Note that this is different that other models from this line of research, because in the alternative scenario we have an additional screening modality, not just increased coverage